src/app/pages/data-protection/components/data-protection-dashboard/data-protection-dashboard.component.ts

```diff
@@ -143,17 +134,12 @@ export class DataProtectionDashboardComponent implements OnInit {
       this.refreshTables();
     });
 
-    this.refreshForms();
-    this.modalService.refreshForm$.pipe(untilDestroyed(this)).subscribe(() => {
-      this.refreshForms();
-    });
-
     this.modalService.message$.pipe(untilDestroyed(this)).subscribe((res: any) => {
       if (res['action'] === 'open' && res['component'] === 'replicationForm') {
-        this.modalService.open('slide-in-form', this.replicationFormComponent, res['row']);
+        this.modalService.openInSlideIn(ReplicationFormComponent, res['row']);
       }
       if (res['action'] === 'open' && res['component'] === 'replicationWizard') {
-        this.modalService.open('slide-in-form', this.replicationWizardComponent, res['row']);
+        this.modalService.openInSlideIn(ReplicationFormComponent, res['row']);
```

Comment:
Missed that piece. Seems like should be ReplicationWizardComponent here

Reply:
Seems so :)